src/app/_components/PortalComponents/Portal/recruitment/RecruitmentTable.tsx

```
@@ -0,0 +1,22 @@
+import { columns } from "./columns";
+
+export type RecruitmentForm = Pick<
+  Recruitment,
+  "id" | "header" | "description" | "link" | "expiresAt"
+>;
+
+const RecruitmentTable = ({ forms }: { forms: RecruitmentForm[] }) => {
+  return (
+    <EntityTable
+      tableHeader="Recruitment"
+      data={forms}
```

ESLint

[error] Props should be sorted alphabetically

(react/jsx-sort-props)